docs/framework/github_feedback.md

```diff
@@ -0,0 +1,175 @@
+
+The feedback interview will save the feedback on the server with or without the above information, so you don't
+have to setup a GitHub account to get feedback from users.
+
+* `default repository owner` should be your main GitHub organization or account that owns most of your interviews. For example, [SuffolkLITLab](https://github.com/SuffolkLITLab), which owns MassAccess's repositories, and [Illinois Legal Aid Online's GitHub account](https://github.com/IllinoisLegalAidOnline).
+* `allowed repository owners` is optional, but necessary if your `default repository owner` doesn't own all of the repositories that you need to provide feedback for. If you want to be able to make
+  issues on repositories from different organizations, you'll need to list each of them here. For example, Suffolk additionally hosts interviews made by [MassLegalHelp](https://github.com/MassLegalHelp). This is so users can't use your tool to spam issues on other organizations GitHub repos.
+* `feedback session linking` helps you as the server administrator debug tricky bugs. If this is set to `True` and if user agrees, the feedback interview will save the session ID of their current interview on the server and link it to their feedback issue on github. This means that you can see the state of their interview when the user ran into their problem, making it easier to reproduce and fix their bug. You can [browse the linked sessions in `browse_feedback_sessions.yml`](#how-to-browse-feedback-on-the-server).
+* `ask panel` will add a question to the feedback interview, asking users if they want to be in a panel for user testing. If the user wants to be in a panel, it will ask for their email and save it in the server. You can also [view these linked sessions in `browse_feedback_sessions.yml`](#how-to-browse-feedback-on-the-server).
+
+### Footer feedback
+
+Next, open the interview that you want to add feedback to in docassemble's playground.
+
+If you include `assembly_line.yml` in your interview already, all you need to do is add the following block to set the `al_github_repo` variable, which `assembly_line.yml` uses to setup the feedback interview correctly.
+
+```yml
+code: |
+  github_repo_name = "YOUR_REPO_NAME" # i.e. "docassemble-MyInterview"
+```
+
+:::tip Duplicate blocks
+
+Use [docassemble's playground search](https://docassemble.org/docs/playground.html#interview_files) to make sure that you only have one block defining the `github_repo_name` variable. If you have more than one, only the last block will be used, which can be confusing.
+:::
+
+If you aren't using `assembly_line.yml`, you can
+add the below two blocks to add a footer to every page of your interview:
+
+```yml
+---
+modules:
+  - docassemble.GithubFeedbackForm.github_issue
+---
+default screen parts:
+  footer: |
+    [:comment-dots: Feedback](${ feedback_link(user_info(), github_repo=YOUR_REPO_NAME) } ){:target="_blank"}
```

Comment:
If we're supporting non-GitHub feedback, then they don't really need anything GitHub-related, right? If so, I think it would be helpful to include how to deal with this if they don't use GitHub. We know at least one team that has a workflow that often didn't (doesn't?) involve GitHub.

Reply:
I think there's a difference between "you can use AssemblyLine without using GitHub at all" and "we're going to help you use AssemblyLine without using GitHub at all". Just because it's possible to do that, doesn't mean that we need to try to document that for them. I get the sentiment, but IMO it's an easy way to get overwhelmed with documentation to write, and doesn't help new users focus on one path through everything.